app/Policies/UserPolicy.php

```diff
@@ -30,7 +30,9 @@ public function viewAny(User $user)
      */
     public function view(User $user, User $model)
     {
-        //
+        return $user->id === $model->id
```

Comment:
return $user->id === $model->id;

remove 
```
? Response::allow()
            : Response::deny('You do not get these data.');
```

we did the same for the update user task, why did you return back to the wrong implementation here?